eng/targets/Settings.props

```diff
@@ -66,6 +66,9 @@
       Related to https://github.com/dotnet/sdk/issues/41791.
     -->
     <BuildWithNetFrameworkHostedCompiler>false</BuildWithNetFrameworkHostedCompiler>
+
+    <!-- https://github.com/dotnet/msbuild/issues/10306 -->
+    <CoreCompileDependsOn>$(CoreCompileDependsOn);ResolveKeySource</CoreCompileDependsOn> 
```

Comment:
Probably don't want to include this change in the PR

Reply:
ah true.  i will remove.  i need this locally @jjonescz  :)